tensorflow-notebook/Dockerfile

```diff
@@ -10,9 +10,7 @@ LABEL maintainer="Jupyter Project <jupyter@googlegroups.com>"
 # Fix: https://github.com/koalaman/shellcheck/wiki/SC3014
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
-# Install Tensorflow
-RUN mamba install --yes \
-    'tensorflow' && \
-    mamba clean --all -f -y && \
+# Install Tensorflow with pip
+RUN pip install tensorflow --no-cache-dir && \
     fix-permissions "${CONDA_DIR}" && \
```

Comment:
Do we need this?

Reply:
Not sure (it was here before me), but we have it after every mamba/pip installation, so let's keep it.